linux/proc/exec.go

```diff
@@ -180,7 +181,8 @@ func (e *execProcess) start(ctx context.Context) (err error) {
 			return errors.Wrap(err, "failed to start console copy")
 		}
 	} else if !e.stdio.IsNull() {
-		if err := copyPipes(ctx, e.io, e.stdio.Stdin, e.stdio.Stdout, e.stdio.Stderr, &e.wg, &copyWaitGroup); err != nil {
+		fifoCtx, _ := context.WithTimeout(ctx, 15*time.Second)
```

Comment:
Value was chosen from 0.2.x branch. We may want to have something much shorter to mitigate the responsiveness problems.